scripts/oauth/token.go

```diff
@@ -0,0 +1,374 @@
+package main
```

Comment:
Uh there's already a script in this directory. Having 2 main methods side-by-side like this is gonna be a problem. Can you rename? Also the script currently in oauth is special-made to obtain oauth tokens. This seems like duplicated functionality.

Reply:
I created the `scripts` directory, you might be thinking of the server?

Reply:
(Also, I copied most of the functionality here from the server script)